draft-ietf-quic-transport.md

```diff
@@ -1483,10 +1483,13 @@ subsequent sections.
 ## STREAM Frame {#frame-stream}
 
 STREAM frames implicitly create a stream and carry stream data. The type byte
-for a STREAM frame contains embedded flags, and is formatted as `11FSSOOD`.
+for a STREAM frame contains embedded flags, and is formatted as `1UFSSOOD`.
 These bits are parsed as follows:
 
-* The first two bits must be set to 11, indicating that this is a STREAM frame.
+* The first bit must be set to 1, indicating that this is a STREAM frame.
+
+* 'U' is the UNI bit, which is used to indicate this is a unidirectional
+  stream, and no stream frames may be sent in response.
```

Comment:
> I'm still confused. Why would the server send on a stream the peer implicitly opened before receiving any data? This seems like a non-problem.

A lot of the complexity with bi-directional streams is that this is possible even if unintended. Adding uni-directional next to bi-directional streams does not simplify things as you now how to check if a reverse frame is valid or not. Uni-directional streams do handle the implicit stream close problem.